monai/apps/auto3d/data_utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+from typing import Dict, Union
+
+__all__ = ["recursive_getvalue", "recursive_setvalue", "recursive_getkey", "datafold_read"]
+
+
+def datafold_read(datalist: Union[str, Dict], basedir: str, fold: int = 0, key: str = "training"):
+    """
+
+    :param datalist: the name of a JSON file listing the data, or a dictionary
```

Comment:
For the format of args doc-string, please follow any other MONAI source code, for example:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/transform.py#L43

Thanks.

Reply:
Thank you Nic for pointing out the issue. I updated the existing functions and will apply the format.